client/components/app-promo/index.jsx

```diff
@@ -69,15 +75,26 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		const element = (
 			<div className="app-promo">
```

Comment:
I wouldn't expect that. [Per the CSS naming guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md), the top-level component in the `render` function of the default export of an `index.js(x)` file should have a class name matching that of the folder it's contained within, which would be correct here. I'll plan to take a closer look soon.